### ISSUE TITLE
ETH + TON TeamRed_v_0.92 on RX580 8gb won't work

### ISSUE THREAD

**Issue by DjUnited27** [todxx/teamredminer#516]
Hi! I just decided to try TeamRed Miner but I face some issues. I set up all needed for dual eth + ton in bat, but got this instead: 
C:\Users\Javad\Desktop\LolMiner\TeamRed\teamredminer-v0.9.2-win\teamredminer-v0.9.2-win>set GPU_MAX_ALLOC_PERCENT=100

C:\Users\Javad\Desktop\LolMiner\TeamRed\teamredminer-v0.9.2-win\teamredminer-v0.9.2-win>set GPU_SINGLE_ALLOC_PERCENT=100

C:\Users\Javad\Desktop\LolMiner\TeamRed\teamredminer-v0.9.2-win\teamredminer-v0.9.2-win>set GPU_MAX_HEAP_SIZE=100

C:\Users\Javad\Desktop\LolMiner\TeamRed\teamredminer-v0.9.2-win\teamredminer-v0.9.2-win>set GPU_USE_SYNC_OBJECTS=1

C:\Users\Javad\Desktop\LolMiner\TeamRed\teamredminer-v0.9.2-win\teamredminer-v0.9.2-win>teamredminer.exe -a ethash -o stratum+tcp://eu1.ethermine.org:4444 -u 0xf53eb0fc3C54b5F11e2a4EC8BE73aeeD00B8D58F.trmtest -p x --ton -o stratum+tcp://pplns.toncoinpool.io:443/stratum -u EQAhFomEhQ-P30NXHDg8etd2VWSxytLjwBhNuxMXqUJOOwhH -p  --ton_end
          Team Red Miner version 0.9.2
[2022-02-11 23:59:50] Pool eu1.ethermine.org: Ethermine ethash auto mode detected, setting --eth_stratum_mode=ethproxy
[2022-02-11 23:59:50] Pool pplns.toncoinpool.io/stratum: using TON pool dialect for toncoinpool.
[2022-02-11 23:59:50] Pool pplns.toncoinpool.io/stratum: enabling wss proxy mode in TRM for toncoinpool.
[2022-02-11 23:59:50] Auto-detected AMD OpenCL platform 0
[2022-02-11 23:59:50] Starting wss local proxy on port 4445 pointing to wss://pplns.toncoinpool.io/stratum, port 443
[2022-02-11 23:59:52] Initializing GPU 0.
[2022-02-11 23:59:52] GPU 0 Passed compute mode and crossfire verification.
[2022-02-11 23:59:52] GPU0 Dual mining kernels are not available (yet) for this gpu. Downgrading to plain eth mining.
[2022-02-11 23:59:52] GPU 0 Starting tuning.
[2022-02-11 23:59:52] Watchdog thread starting.
[2022-02-11 23:59:52] Enabling 2nd algo (ton)
[2022-02-11 23:59:52] Successfully initialized GPU 0: Polaris with 32 CU (PCIe 01:00.0)
[2022-02-11 23:59:52] Pool pplns.toncoinpool.io/stratum (PROXY) connecting to address 127.0.0.1.
[2022-02-11 23:59:52] Runtime Command Keys: h - help, s - stats, e - enable gpu, d - disable gpu, q - quit
[2022-02-11 23:59:52] Wss proxy: Host 'pplns.toncoinpool.io' url '/stratum'
[2022-02-11 23:59:52] Wss proxy: Accepting connections on 0.0.0.0:4445
[2022-02-11 23:59:52] Pool eu1.ethermine.org connecting to address 172.65.207.106.
[2022-02-11 23:59:52] Pool eu1.ethermine.org successfully connected to address 172.65.207.106.
[2022-02-11 23:59:52] Pool eu1.ethermine.org login succeeded.
[2022-02-11 23:59:53] Pool pplns.toncoinpool.io/stratum (PROXY) successfully connected to address 127.0.0.1.
[2022-02-11 23:59:53] Wss proxy: [1] ::ffff:127.0.0.1:49939 - Session 1 connected.
[2022-02-11 23:59:53] Pool eu1.ethermine.org set difficulty to 1.000 (4295 MH)
[2022-02-11 23:59:53] Pool eu1.ethermine.org received new job. (job_id: 0x96a58e4d2ae..., diff 1.000 / 4295 MH)
[2022-02-11 23:59:53] Dev pool connected and ready.
[2022-02-11 23:59:53] Dev pool connected and ready.
[2022-02-11 23:59:53] Wss proxy: [1] ::ffff:127.0.0.1:49939 - pool proxy connection established.
[2022-02-11 23:59:53] Pool pplns.toncoinpool.io/stratum (PROXY) successfully subscribed. (566 ms)
[2022-02-11 23:59:53] Pool pplns.toncoinpool.io/stratum (PROXY) authorization succeeded.
[2022-02-11 23:59:53] Pool pplns.toncoinpool.io/stratum (PROXY) received new job. (job_id: 121c8cfe8d45de30)
[2022-02-11 23:59:57] Pool eu1.ethermine.org received new job epoch 472 (variant ethash), cache built in 3.8 secs.
[2022-02-11 23:59:57] Pool eu1.ethermine.org received new job. (job_id: 0x5ff9687dcd2..., diff 1.000 / 4295 MH)
[2022-02-11 23:59:57] Pool eu1.ethermine.org received new job. (job_id: 0x99788dfa899..., diff 1.000 / 4295 MH)
[2022-02-11 23:59:57] GPU 0 Pause hashing (ton).

So I'll really glad to see any possible suggestion to solve it.

**Comment by affer** [comment]
TL;DR

To mine ETH+TON on Navi gpus (which is the only supported dual algo at the moment),
source: https://github.com/todxx/teamredminer/blob/master/doc/DUAL_ETH_MINING.txt

FYI RX580 is NOT a Navi gpu
